.github/workflows/build-test-windows.yml

```diff
@@ -0,0 +1,33 @@
+
+name: build-and-test-windows
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Do we do version tags such that we would trigger this Action on a tag?

Reply:
copypasta